### ISSUE TITLE
Variable not defined in Polyester v0.4.4

### ISSUE THREAD

**Issue by ranocha** [JuliaSIMD/Polyester.jl#48]
We get the [following error in CI](https://github.com/trixi-framework/Trixi.jl/runs/3563782783#step:7:2776) using Polyester v0.4.4:
```
elixir_euler_weakform.jl (SBP, EC): Error During Test at /home/runner/work/Trixi.jl/Trixi.jl/test/test_trixi.jl:209
  Got exception outside of a @test
  LoadError: UndefVarError: i not defined
  Stacktrace:
    [1] macro expansion
      @ ~/work/Trixi.jl/Trixi.jl/src/solvers/dgmulti/dg.jl:108 [inlined]
[ Info: Testset elixir_euler_weakform.jl (SBP, EC) finished in 32.617338419 seconds.
WARNING: replacing module TrixiTestModule.
    [2] #625
      @ ~/.julia/packages/Polyester/7SKjv/src/closure.jl:224 [inlined]
    [3] macro expansion
      @ ~/.julia/packages/Polyester/7SKjv/src/batch.jl:81 [inlined]
    [4] _batch_no_reserve
      @ ~/.julia/packages/Polyester/7SKjv/src/batch.jl:52 [inlined]
    [5] batch
      @ ~/.julia/packages/Polyester/7SKjv/src/batch.jl:218 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/Polyester/7SKjv/src/closure.jl:289 [inlined]
    [7] macro expansion
      @ ~/work/Trixi.jl/Trixi.jl/src/auxiliary/auxiliary.jl:188 [inlined]
```
The relevant code is
```julia
  @threaded for i in each_quad_node_global(mesh, dg, cache)
    u_values[i] = initial_condition(getindex.(md.xyzq, i), t, equations)
  end
```
where `@threaded` redirects everything directly to `@batch`.

**Comment by chriselrod** [comment]
This has already been fixed on 0.5.0.

**Comment by ranocha** [comment]
Okay, thanks. Then we will wait until updated compat bounds have propagated through the ecosystem
